fuel-poa-coordinator/src/service.rs

```diff
@@ -174,11 +176,30 @@ where
         }
 
         // Ask the block producer to create the block
-        let block = self.signal_produce_block().await?;
+        let ExecutionResult {
+            block,
+            skipped_transactions,
+        } = self.signal_produce_block().await?;
 
         // sign the block and seal it
         self.seal_block(&block)?;
 
+        let mut tx_ids_to_remove = Vec::with_capacity(skipped_transactions.len());
+        for (tx, err) in skipped_transactions {
+            error!(
```

Comment:
These should probably not be `error` level as this isn't an error with the software. Maybe `info` level?

Reply:
If everything works correctly, we shouldn't have cases with skipped transactions at all. It is why I marked it `error`=)